4-star-nosed_mole/api_definitions/rest/drs3.yaml

```diff
@@ -0,0 +1,225 @@
+              schema:
+                type: integer
+                format: int64
+          content: {}
+        400:
+          description: The request is malformed.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+        401:
+          description: The request is unauthorized.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+        403:
+          description: The requester is not authorized to perform this action.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+        404:
+          description: The requested `DrsObject` wasn't found
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+        500:
+          description: An unexpected error occurred.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+      x-swagger-router-controller: ga4gh.drs.server
+components:
+  schemas:
+    Checksum:
+      required:
+        - checksum
+        - type
+      type: object
+      properties:
+        checksum:
+          type: string
+          description: The hex-string encoded checksum for the data
+        type:
+          type: string
+          description:
+            The digest method used to create the checksum. The value (e.g.
+            `sha-256`) SHOULD be listed as `Hash Name String` in the https://www.iana.org/assignments/named-information/named-information.xhtml#hash-alg[IANA
+            Named Information Hash Algorithm Registry]. Other values MAY be used,
+            as long as implementors are aware of the issues discussed in https://tools.ietf.org/html/rfc6920#section-9.4[RFC6920].
+            GA4GH may provide more explicit guidance for use of non-IANA-registered
+            algorithms in the future. Until then, if implementors do choose such an
+            algorithm (e.g. because it's implemented by their storage provider), they
+            SHOULD use an existing standard `type` value such as `md5`, `etag`, `crc32c`,
+            `trunc512`, or `sha1`.
+          example: sha-256
+    DrsObject:
+      required:
+        - checksums
+        - created_time
+        - id
+        - self_uri
+        - size
+        - access_methods
+        - name
+      type: object
+      properties:
```

Comment:
There are some properties missing defined in the drs spec:
updated_time and version make no sense for our use case, I guess.
What about aliases, description and mime_type?

Reply:
We currently don't use them either. All of these information would be queriable via our metadata services. It might be a bit redundant and create extra work to repeat it here. But at a later stage we could re-evaluate.